tueplots/axes.py

```diff
@@ -36,9 +37,10 @@ def lines(
         "grid.color": color,
         # Update the linestyle of the grid
         # (it shares a color with the frame, and needs to be distinguishable)
-        "grid.linestyle": "dotted",
+        "grid.linestyle": "solid",
```

Comment:
Since it seems that this setting is opinionated, how about we make it an optional argument to the function. The default can be "solid", if you dislike dotted lines.

I think this solution would be preferable, because it does not impose anything on users, but merely suggests some settings.

Reply:
Thanks, I agree with this!